src/lib/link-meta/resolve-short-link.ts

```diff
@@ -0,0 +1,23 @@
+import {logger} from '#/logger'
+import {startUriToStarterPackUri} from 'lib/strings/starter-pack'
+
+export async function resolveShortLink(shortLink: string) {
+  const controller = new AbortController()
+  const to = setTimeout(() => controller.abort(), 2e3)
```

Comment:
these _should_ resolve in two seconds, but we can bump to maybe 5 if we need to.